questions.md

```diff
@@ -0,0 +1,4 @@
+#Questions
+
+* What's the best way to help students remember all the commandline syntax? 
```

Comment:
Hi @nicschwartz ! The trick is really practice. The more they use the commands the better they will be. I would recommend an assignment or project where you ask students to commit and commit often. That way they get use to the commands and working through the working directory, staging, and the local repository.